src/web/lib/hakuneko/frontend@classic-light/mangas.html

```diff
@@ -262,11 +296,41 @@
                 //
             }
 
+            /**
+            * Event listener attached to the refresh button
+            * Checks for new content to read by refreshing the bookmarks, updating the web source and checking the chapters bookmarks
+            */
+            async onCheckNewContent() {
+                if(! this.get('enableCheckNewContent'))
+                {
+                    alert('Please wait 60s before checking again.');
+                    return;
+                }
+                this.set('nbContentUpdating',Engine.BookmarkManager.bookmarks.length); 
+                await Engine.BookmarkManager.checkNewContent();
```

Comment:
I don't understand the 60 second timer.
The content check is awaitable, so after this is done the flag can just be set to false directly without using the timer?